### ISSUE TITLE
Adding terminology for indicating expected and actual node resolution

### ISSUE THREAD

**Issue by gaurav** [phyloref/phyloref-ontology#55]
In Klados PR https://github.com/phyloref/klados/pull/313, we implement the following four Nexus properties:
* `phyloref:expected="[URI]"`: The phyloreference expected to resolve at this node (e.g. `http://example.org/phyloref/Crocodylidae`).
* `phyloref:expectedLabel="[label]"`: The label of the phyloreference expected to resolve at this node (e.g. `Crocodylidae`).
* `phyloref:actual="[URI]"`: The phyloreference that actually resolved at this node (e.g. `http://example.org/phyloref/Crocodylidae`).
* `phyloref:actualLabel="[label]"`: The label of the phyloreference that actually resolved at this node (e.g. `Crocodylidae`).

For example, a node could be annotated as:

```
(A, B)C[&phyloref:expected="#Crocodylidae",phyloref:expectedLabel="Crocodylidae",phyloref:actual="#Crocodylidae",phyloref:actualLabel="Crocodylidae"]
```

In the example file in this repository, we use [obo:RO_0002558 "has evidence"](http://purl.obolibrary.org/obo/RO_0002558) to connect a phyloreference to its expected or resolved node. For example:

https://github.com/phyloref/phyloref-ontology/blob/2fc7cfcee0dbeb4f06a45d28a665c96dd4ab0bb5/gators.ofn#L189-L200

So we could use the reverse relation ([obo:RO_0002472 "is evidence for"](http://purl.obolibrary.org/obo/RO_0002472)) to connect a node with their phyloreference. However, using a Nexus property of `RO:0002472` or `obo:RO_0002472` might be unclear in this context, and we would still need to distinguish between the expected resolution node and the actual resolution node.

I think our options are:
1. Go with `RO:0002472` for the actual resolved node, and denote the expected node in the label (e.g. `expected_Crocodylidae`)
2. Come up with additional properties for these nodes (such as `phyloref:expected` etc. listed above). We would define them as annotation properties, with ranges of IRIs for `phyloref:expected`/`phyloref:actual` and strings for `phyloref:expectedLabel`/`phyloref:actualLabel`.
3. ~We could use a URL as the property name (e.g. `[&http://ontology.phyloref.org/expectedPhyloref="http://example.org/phyloref/Crocodylidae"]`) and have that URL resolve to a website that explains what it means for humans, and tells people how to represent this in RDF.~

**Comment by gaurav** [comment]
- [ ] We need the properties to be fully understandable without the prefix, since that might not be visible in RDF apps. So something like e.g. `phyloref:expectedPhyloref`.

**Comment by gaurav** [comment]
- [ ] We are ready to implement this as a PR for review.

**Comment by hlapp** [comment]
I agree there's something to be said for having a non-opaque property for exporting NEXUS tree annotations. I'm not sure these need or should be annotation properties. More importantly, we can make them (or at least  `expectedPhyloref`) a subproperty of the RO property.

For the "actual" pair of properties, this may deserve a little more thought. I don't think we really mean to say "this [node] is the actual evidence for this [phyloreference]", but more like "this phyloreference resolves to this node", or conversely, "this node is to where this phyloreference resolves". So if we can agree that "resolves to X" and "matches X" are sufficiently close semantically, we could use that?
